### DISCUSSION TITLE
KabelExtra for R4.4

### DISCUSSION THREAD

**Discussion by ScottWrightPHW** [haozhu233/kableExtra#850] [Q&A]
Hi.  Does anyone know if there is a version of KableExtra for r4.4?  I've tried searching and come up with nothing.  Any help appreciated

**Comment by dmurdoch** [comment]
Yes, but the name is "kableExtra" not "KableExtra".

**Reply by ScottWrightPHW** [reply]
Thanks.  That's what I meant.  Funny enough, I tried again and the package installed from the CRAN and worked.  So no panic now.  Thanks for your reply but it appears the version which worked previously still works under v4.4 of R.
